packages/bvaughn-architecture-demo/src/hooks/useBreakpointIdsFromServer.ts

```diff
@@ -53,10 +51,11 @@ export default function useBreakpointIdsFromServer(
         prevPoints.forEach(prevPoint => {
           const point = prevPoints.find(({ id }) => id === prevPoint.id);
```

Comment:
this should be `points.find(...)`, otherwise `point` will never be `undefined` and breakpoints will not be removed on the server.

Reply:
lol yeah that's a silly mistake on my part. You're right.